Mithril update 2024-08-07 (#440)
IntersectMBO/cardano-updates: blog/2024-08-07-mithril.md

---
title: Mithril Team Update
slug: 2024-08-07-mithril
authors: jpraynaud
tags: [mithril]
hide_table_of_contents: false
---

## High level overview

> This week, the Mithril team continued their work on the certification of Cardano stake distribution: they completed the implementation of new routes for the aggregator REST API and worked on the implementation of the verification steps in the client library. They also almost completed the draft CIP for the diffusion of Mithril signatures through the Cardano network. Additionally, they completed the benchmarking of the signer's footprint for the Cardano transactions on the SPO infrastructure.

Reviewer: ```suggestion
This week, the Mithril team continued their work on the certification of Cardano stake distribution. They completed the implementation of new routes for the aggregator REST API and worked on the verification steps in the client library. The draft CIP for the diffusion of Mithril signatures through the Cardano network is nearly complete. Additionally, they finished benchmarking the signer's footprint for Cardano transactions on the SPO infrastructure.
```

> Finally, the team fixed a bug on the transaction importer which prevented some transactions to be imported when working close to the tip of the chain, and a they fixed a bug in the documentation website.

Reviewer: ```suggestion
The team also fixed a bug in the transaction importer that prevented some transactions from being imported near the chain tip and resolved an issue on the documentation website
```
